lib/src/parse/stylesheet.dart

```diff
@@ -805,6 +803,7 @@ abstract class StylesheetParser extends Parser {
   ///
   /// [start] should point before the `@`.
   DebugRule _debugRule(LineScannerState start) {
+    whitespace(allowNewlines: false);
```

Comment:
Because this is a Sass at-rule that's always followed by an expression, we should allow newlines after the name.